### PR TITLE
add rabbit chain

### PR DESCRIPTION
**Pull request by amankumarp**
Added Rabbit Analog Chain.

Chain ID:1807
RPC Endpoint: https://rabbit.analog-rpc.com
Block Explorer: https://rabbit.analogscan.com

https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-1807.json

### CODE REVIEW

**Comment by moisses89** [comment]
@amankumarp could you create an issue for new chain and link the PR? thanks

File: src/assets/v1.3.0/sign_message_lib.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "1285": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "1287": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "1294": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+    "1807": "0xa12FbcAC4ee9541fc7Bd3B9A4A6E2eD080B0ee41",
```
[the remark this answers is not in the document]
Comment (amankumarp) [reply src/assets/v1.3.0/sign_message_lib.json:47]: yes i am using v1.3.0-libs.0

[the remark this answers is not in the document]
Comment (moisses89) [reply src/assets/v1.3.0/sign_message_lib.json:47]: Something is wrong this address is not the deterministic address. For example take a look like for each contract except this one the address is the same than the address for the chain 1984, could you try to run again the deploy and check the address again?

[the remark this answers is not in the document]
Comment (amankumarp) [reply src/assets/v1.3.0/sign_message_lib.json]: updated signMessageLib Address please check

**Review by moisses89 — APPROVED** [review]
Looks good to me!

**Pull request merged**